JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2253,12 +2238,10 @@
   n_bad_loo  <- length(loo::pareto_k_ids(loo, threshold = .7))
```

Comment:
In general, I'd avoid snake_casing. Is there a specific reason to do so?

Reply:
no, I just find it much more readable when writing code

Reply:
Why sometimes snake_casing and sometimes camelCasing? There's a preference for camelCasing in jasp code.

Reply:
changed